api/src/repositories/survey-critter-repository.ts

```diff
@@ -43,6 +43,16 @@ export class SurveyCritterRepository extends BaseRepository {
     return response.rowCount;
   }
 
+  async updateCritter(surveyId: number, critterId: string): Promise<number> {
+    defaultLog.debug({ label: 'updateCritter', surveyId });
+    const queryBuilder = getKnex()
+      .table('critter')
+      .update({ survey_id: surveyId, critterbase_critter_id: critterId })
+      .where({ survey_id: surveyId, critterbase_critter_id: critterId });
```

Comment:
This query doesn't make a lot of sense to me. The `WHERE` predicate matches the body of the update, so nothing is actually being modified... Unless I'm missing something?

Reply:
Yeah, it's effectively just a placeholder, since the critter table on SIMS end doesn't really hold much else of note. Doing the update operation will fill in the update audit columns at least.

Reply:
OK gotcha. Might make sense to leave a comment if we're gonna leave it like this for now